scripts/docker-utils.sh

```
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+remove_docker_container() {
+  local container_name="$1"
+
+  echo "\nRemoving existing container: $container_name"
```

Shellcheck (0.10.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)